games/c.yaml

```diff
@@ -614,11 +615,11 @@
   content: free
   originals:
   - System Shock
-  status: unplayable
+  status: playable
   repo: https://github.com/JosiahJack/Citadel
-  development: sporadic
+  development: active
   type: remake
-  updated: 2022-01-30
+  updated: 2023-04-15
```

Comment:
```suggestion
  updated: 2023-05-11
```

Reply:
Please note that this date is meant to reflect when the game was updated in OSGC, not the game project